packages/devextreme/js/__internal/ui/m_select_box.ts

```diff
@@ -427,6 +427,26 @@ const SelectBox = (DropDownList as any).inherit({
     return this._dataController.searchValue();
   },
 
+  _getAriaAutocomplete() {
+    const {
+      disabled,
+      readOnly,
+      searchEnabled,
+      searchMode,
+    } = this.option();
+
+    const isAvailable = !(readOnly || disabled);
+    const hasAutocomplete = searchEnabled && isAvailable;
+
+    if (!hasAutocomplete) {
+      return 'none';
+    }
+
+    const autocompleteAria = searchMode === 'contains' ? 'list' : 'both';
```

Comment:
I also see that inline autocomplete is not supported when acceptCustomValue=true.

I suggest u make a separate function `isInlineAutocompleteEnabled` checking that 
```javascript
            this.option('searchEnabled')
            && !this.option('acceptCustomValue')
            && this.option('searchMode') === 'startswith';
 ```
, use it here and inside of `_shouldSubstitutionBeRendered` to reduce copypaste.
 
 Don't forget to add a test as well.

Reply:
[623a2f1e37e18a68cb03cafce4131873b3afc72a]